### DISCUSSION TITLE
Hello, can you add a feature to share notes in the form of images?

### DISCUSSION THREAD

**Discussion by james0336** [damionx7/Zettel-Notes-Documentation#191] [Feature Requests]
Hello, can you add a feature to share notes in the form of images? Sometimes sharing notes with friends, images are more convenient than PDF.

**Comment by damionx7** [comment]
Hey! Will the note be shared in multiple images or a single image. And can you suggest any apps that already implement this feature.

**Comment by james0336** [comment]
This app is capable of doing that
https://play.google.com/store/apps/details?id=com.colanotes.android

**Comment by damionx7** [comment]
I have added this feature. Its live in the latest update.
